### PR TITLE
feat: add activity chart

### PR DESCRIPTION
**Pull request by berteotti**
Screenshot 2024-06-27 at 14 45 53

### CODE REVIEW

**Review by ElRodrigote — APPROVED** [review]

File: app/markets/ActivityChart.tsx [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+  const TooltipContent = (chartData: DefaultTooltipContentProps<number, string>) => {
+    const { payload } = chartData;
+
+    if (!payload || !payload[0] || !payload[1]) return;
+
+    const total = payload.reduce(
+      (result, entry) => (entry.value ? result + +entry.value : 0),
+      0
+    );
+    const tradeTime = format(fromUnixTime(+payload[0].payload.time), 'HH:mm dd/MM/yy');
```
Comment (Diogomartf) [inline app/markets/ActivityChart.tsx:109]: should we use a fotmat from `utils/date` or if doesn't existe extracting there?

File: app/markets/ActivityChart.tsx [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+            .sort((a, b) => {
+              if (!a.dataKey || !b.dataKey) return 0;
+              return +a.dataKey - +b.dataKey;
+            })
+            .map((entry, index) => (
+              <li
+                key={index}
+                className={cx({
+                  'text-text-danger-em': entry.dataKey === '0',
+                  'text-text-success-em': entry.dataKey === '1',
```
Comment (Diogomartf) [inline app/markets/ActivityChart.tsx:125]: what means '0' and '1'? That's a great opportunity to use a constant to describe this value.

An example
```suggestion
                  'text-text-danger-em': entry.dataKey ===  OUTCOME_NO,
                  'text-text-success-em': entry.dataKey === OUTCOME_NO,
```

File: app/markets/ActivityChart.tsx [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+          bottom: 0,
+        }}
+      >
+        <Tooltip content={TooltipContent} />
+        <Area
+          type="monotone"
+          dataKey="1"
+          name={marketModel.outcomes[0].symbol}
+          stackId="1"
+          fill={resolvedTheme === 'light' ? '#AFFAD0FF' : '#134228'}
```
Comment (Diogomartf) [inline app/markets/ActivityChart.tsx:160]: Suggestion:
`const isLightheme = resolvedTheme === 'light'`

```suggestion
          fill={isLightheme ?  '#AFFAD0FF' : '#134228'}
```

**Review by Diogomartf — APPROVED** [review]

File: app/markets/ActivityChart.tsx (hunk 1 above)
Comment (berteotti) [reply app/markets/ActivityChart.tsx:124]: I think this can live here for now.

File: app/markets/ActivityChart.tsx (hunk 2 above)
Comment (berteotti) [reply app/markets/ActivityChart.tsx]: Change to OUTCOME_0 and OUTCOME_1, there are markets with different outcome names

**Pull request merged**